en/asgardeo/docs/tutorials/smart-on-fhir.md

```diff
@@ -0,0 +1,256 @@
+                    <li>Delete Claim</li>
+                    <li>Update Claim</li>
+                    <li>Create Claim</li>
+                    <li>View Claim</li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>API Resource Management API</td>
+            <td>
+                <ul>
+                    <li>Delete API Resource</li>
+                    <li>Update API Resource</li>
+                    <li>Create API Resource</li>
+                    <li>View API Resource</li>
+                </ul>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+To do so,
+
+1. On the [Asgardeo Console](https://console.asgardeo.io/), go to **Applications**.
+
+2. Select the created application and go to its **API Authorization** tab.
+
+3. Do the following for all the REST APIs listed on the above table.
+
+    1. Click on **Authorize an API Resource** and do the following:
+    
+    2. Select the resource from the **API Resource** dropdown.
+
+        !!! note
+
+            Be sure to select the API resources listed under **Management API**.
+
+    2. Under **Authorized Scopes**, select the relevant scopes. Since the application requires all scopes of the API resource, click **Select All** to collectively add all scopes.
+
+    3. Click **Finish**.
+
+    ![Authorize app to consume API]({{base_path}}/assets/img/tutorials/smart-on-fhir/fhir-authorize-app-for-api.png){: width="600" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+### Step 3: Configure the Postman collection
+
+Now that you have registered and configured an application in Asgardeo, you are able to use its credentials to obtain an access token and access Asgardeo's REST APIs.
+
+We have created a Postman collection to automate the following process:
+
+- Get a bearer token to access Asgardeo REST APIs
+
+- Create the `fhirUser` user attribute. This will be used as the identity of a user accessing FHIR resources.
+
+- Create the `fhirUser` scope. Applications can request this scope to access the `fhirUser` attribute of a user.
+
+- Add the relevant OIDC and SCIM dialects of the `fhirUser` user attribute.
+
+- Update the `fhirUser` attribute of a selected user to `Patient/1`.
+
+- Register your SMART on FHIR app using Dynamic Client Registration (DCR).
+
+- Configure your SMART on FHIR application to request the `fhirUser` local attribute from users during login.
+
+Follow the steps below to download and run the Postman collection:
+
+1. Download the Postman collection from [Github](https://github.com/wso2-enterprise/open-healthcare/blob/main/scripts/postman-collections/smartonfhir-asgardeo.postman_collection.json){target="_blank"} and import it to Postman.
+
+2. In the Postman collection named **Asgardeo**, go to the **Authorization** tab and do the following:
+
+    1. Under **Auth Type**, select **Basic Auth**.
+
+    2. Enter the client ID and client secret of your application as username and password respectively.
```

Comment:
Shall we make the client id and client secret values not visible?

Reply:
Also shall we provide a screenshot of the application created in Asgardeo after running the Postman Collection?

Reply:
Will add the SS of the API authorization tab